index.html

```diff
@@ -0,0 +1,237 @@
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_04.png" alt="icon" />
+            <h3 class="program-text1">Workshop</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_05.png" alt="icon" />
+            <h3 class="program-text1">CC Ignite</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum.
+            </p>
+          </div>
+          <a href="#" class="join-btn">Join CC Global Submit 2015</a>
+        </div>
+        <a href="#" class="program-link">SEE THE WHOLE PROGRAM</a>
+      </section>
+
+      <!-- Speakers -->
+      <section class="speakers">
+        <h2>featured Speakers</h2>
+        <hr class="line1" />
+        <div class="speakersContainer">
+        </div>
+        <button class="btn more">
+          <a href="#">MORE</a>
+          <img src="./img/arrow_down.png" alt="arrow icon" />
+        </button>
+        <a href="#" class="btn less">LESS</a>
+      </section>
+    </div>
+
+    <!-- menu -->
+    <section class="menu-click">
+      <div class="close-container">
+        <img class="close-icon" src="./img/cross-icon.png" alt="close icon" />
+      </div>
+      <ul class="menu-text">
+        <li>
+          <a href="#Portfolio" class="nav-link">Portfolio</a>
+          <hr class="menu-line" />
+        </li>
+        <li>
+          <a href="#about" class="nav-link">About</a>
+          <hr class="menu-line" />
+        </li>
+        <li>
+          <a href="#contact-form" class="nav-link">Contact</a>
+          <hr class="menu-line" />
+        </li>
+      </ul>
+    </section>
+
+    <!-- page 2 -->
+    <div class="page2 about-page">
+      <!-- about -->
+      <section class="about">
+        <h3 class="sub-title">"Hello! sharing the world!"</h3>
+        <h1 class="about-title">
+          WORLD CONFerance <br />
+          ALERTS 2022
+        </h1>
+        <p class="paragraph1 color">
+          Full-stack Web Development courses offered through Coursera equip learners with knowledge in building web, hybrid, and mobile solutions; understanding front-end frameworks such as Bootstrap 4 and Angular; implementing NoSQL databases using MongoDB; creating the best user
+           experience regardless of device; the basics of ...
+        </p>
+        <h2 class="contact">
+          Please contact us per Email for any further questions<br />
+          about CC Global Summi 2015!<br />
+          <a href="#">ccsummit@cckorea.org</a>
+        </h2>
+      </section>
+
```

Comment:
- [ ] On the mobile version this section is overlapping the 'Global Summit' section. Kindly fix the styling to make a better user experience.
 - [ ] It is also missing in the 'About' page the middle section containing the logo of the page on both mobile and desktop versions. Kindly implement this section to meet the template.
 - Your mobile version:
 - 
<img width="442" alt="image" src="https://user-images.githubusercontent.com/61919617/210401038-a041c272-67fb-46ea-a300-2efbefbe136f.png">

 - Template mobile version:
 - 
<img width="569" alt="image" src="https://user-images.githubusercontent.com/61919617/210401164-1ec9e741-1200-466c-b986-21826bebd6df.png">

 - Your desktop version:
 - 
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/61919617/210401276-8ac313ac-144e-45e5-8c97-d8a132d1833d.png">

 - Template desktop version:
 - 
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/61919617/210401393-e27591b7-c11f-41cf-a53a-39114d9e9ada.png">

Reply:
done